Sources/System/FilePath/FilePathComponents.swift

```diff
@@ -166,15 +176,22 @@ extension _StrSlice {
   internal var _systemString: SystemString { SystemString(_slice) }
 }
 extension _StrSlice {
-  public static func == (lhs: Self, rhs: Self) -> Bool {
+  public static func strSliceEqual(lhs: Self, rhs: Self) -> Bool {
```

Comment:
If we want to go this way, let's at least put an end to adding any new `public` members of internal types. That only leads to sadness.

```suggestion
  internal static func strSliceEqual(lhs: Self, rhs: Self) -> Bool {
```